submissions/description/application_level_denial_of_service_dos/critical_impact_and_or_easy_difficulty/guidance.json

```diff
@@ -0,0 +1,6 @@
+{
+  "overview": "Provide a 1-2 sentence description - see [our key details phrasing guide](http://cveproject.github.io/docs/content/key-details-phrasing.pdf) for tips. This format is a good guide: [VULNTYPE] in [COMPONENT] in [APPLICATION] allows [ATTACKER] to [IMPACT] via [VECTOR]",
```

Comment:
Note that here I've removed line breaks.

It might be desirable to allow some spacing of sections? If so, this should probably be an array containing two fairly short strings, rather than one longer string.